spec/support/factory_girl.rb

```diff
@@ -21,7 +21,7 @@
     longitude 116.40739499999995
     latitude 39.904211
     is_achieved false
-    notes 'An disturbingly tiny bar on the edge of the Forbidden City'
+    notes 'Do NOT order baijiu'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.